lib/index.ts

```diff
@@ -329,7 +329,7 @@ class Romcal {
   static LOCALE_IDS = LOCALE_IDS;
 }
 
-export default Romcal;
+export { Romcal };
```

Comment:
@emagnier, why don’t we simply export `Romcal` at its definition? :thinking: See my suggestion there.

```suggestion
```

Reply:
Technically, it's the same thing. Here, it's just a matter of readability convenience: all the `export` are gathered in one place in this file.

Reply:
I agree with you it is the same, however, for my point of view it is a useless line and extra 17 characters (` default Romcal;` plus a newline). I personally prefer exporting stuff where they are defined instead of exporting them separately.

As for the _readability convenience_, I personally see no advantage, as:

- IMHO developers use IDEs which use intellisense to gather all the info and has many _Go to_ features;
- even for other users, I believe that searching a file for `export` word is not that hard.

YMMV. If you disagree with me, feel free to merge the PR.

Reply:
I agree with your point of view, except for this specific case, for the reasons I've just mentioned. I prefer not to overlook the developer experience, rather than save characters (for code that is transpiled anyway).